common/djangoapps/course_modes/views.py

```diff
@@ -26,6 +26,8 @@
 from openedx.core.djangoapps.embargo import api as embargo_api
 from student.models import CourseEnrollment
 from util.db import outer_atomic
+from util.enterprise_helpers import get_enterprise_learner_data, enterprise_enabled
```

Comment:
I'd love to see us shift to a pattern similar to organizations_helpers where we work with the module as `enterprise_api`